app/assets/stylesheets/panamax/images.css.scss

```diff
@@ -69,10 +70,24 @@ ul.images li {
   border-bottom: $light_grey 1px solid;
   overflow: auto;
   padding-bottom: 22px;
+  padding-top: 22px;
+
+  .image-layers-link {
+    background: image-url('icon_image_layers.svg') -10px -17px;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Prefer double-quoted strings